### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by bhtibrewal**
In This pr:
- layout the homepage 
- added login modal
- subscribe modal
- collection section

### CODE REVIEW

File: pages/homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,162 @@
+
+/*  */
+.arstyShop-header {
```
Comment (varun1233) [inline pages/homepage/homepage.css:3]: trying using snake-case for variable naming in css.

**Review by purohitdheeraj — APPROVED** [review]
Code Looks great for merge
for smaller screen buttons are sticking 
you can simply wrap it or add some gap between them

File: pages/homepage/homepage.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const loginSection = document.querySelector(".login-section-overlay");
+
+const closeFunc = (e) => {
+    e.parentElement.style.display = "none";
+    console.log("clicked")
+}
+const openLogin=()=>{
+    loginSection.style.display="flex";
+    console.log("click")
```
Comment (varun1233) [inline pages/homepage/homepage.js:9]: Remove Consoles

File: pages/homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    width: 50%;
+    max-width: 54rem;
+    padding: 4rem;
+    align-items: center;
+    gap: 3rem;
+    background: var(--light-bg);
+    text-align: center;
+  }
+  
+  /*  */
```
Comment (varun1233) [inline pages/homepage/homepage.css:129]: Remove comments where not required

File: pages/homepage/homepage.css (hunk 1 above)
Comment (bhtibrewal) [reply pages/homepage/homepage.css:3]: okay

File: pages/homepage/homepage.js (hunk 2 above)
Comment (bhtibrewal) [reply pages/homepage/homepage.js:9]: okay

File: pages/homepage/homepage.css (hunk 3 above)
Comment (bhtibrewal) [reply pages/homepage/homepage.css:129]: okay

**Comment by gauravsinhaweb** [comment]
Looks good to me, nice work Bhavika! 👍

**Review by gauravsinhaweb — APPROVED** [review]
Looks good to me, nice work Bhavika! 👍

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks nice , if possible try to overlap the counts of badges at the top of icons ..rest all is good

**Pull request merged**